src/components/atoms/textInput/InputText.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './InputText.css';
+
+export default function InputText(props) {
+  return (
+    <input className="input" type="text" placeholder={props.placeholder} ref={props.searchName}/>
+  );
+}
```

Comment:
그럼 이부분도 이렇게 수정되어야할듯합니다
```suggestion
export default function InputText(props) {
  const { placeholder, onChange, value } = props;

  return (
    <input className="input" type="text" placeholder={placeholder} onChange={onChange} value={value} />   
  );
}
```

Reply:
아하....! ref를 직접 전달하지 않아도 되는건가요...!

Reply:
ref를 전달한다기보단, onChange 이벤트 걸어서 값을 보낸다고 생각하시면 이해하시기 편하실듯합니다!

Reply:
아하! 감사합니다 ㅎㅎㅎ